### PR TITLE
Add tests dependencies

### PR DESCRIPTION
**Pull request by michaelloewenstein**

### CODE REVIEW

File: requirements-dev.txt [hunk 1]
```diff
@@ -0,0 +1,3 @@
+tox==3.20.0
```
Comment (avara1986) [inline requirements-dev.txt:1]: Thank you @michaelloewensteinviz ! Could be great that the `requirements-dev.txt` had this at first line `-r requirements.txt` to install dev dependencies and the rest of dependencies at once

**Review by avara1986 — CHANGES_REQUESTED** [review]

File: requirements-dev.txt (hunk 1 above)
Comment (michaelloewenstein) [reply requirements-dev.txt:1]: @avara1986 , not sure what you mean, having all the dependencies in a single file (requirements.txt) ?

Comment (avara1986) [reply requirements-dev.txt:1]: You can see [this file](https://github.com/avara1986/pivoandcode-2019-11-15/blob/master/chat_front/requirements-docker.txt) as example of what a I mean :) 

If you add in your file `requirements-dev.txt` this: `-r requirements.txt` when you type `pip install -r requirements-dev.txt` pip will install requirements-dev AND requirements at once

Comment (michaelloewenstein) [inline requirements-dev.txt:1]:
```suggestion
-r requirements.txt
tox==3.20.0
```

Comment (michaelloewenstein) [reply requirements-dev.txt:2]: Good point! added

**Review by avara1986 — APPROVED** [review]

**Pull request merged**